src/utils/read_from_file/handle_tutorial_battle_cmd.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "./handle_battle_cmd.h"
+#include "../../creature/main_character/main_character.h"
+#include "../../creature/main_character/main_character_prototypes.h"
+
+#include <fstream>
+#include <functional>
+#include <string>
+
+namespace utils
+{
+/**
+ * @brief This function handles the "BATTLE" command from `ReadFromFile()`.
```

Comment:
This is a copy-paste from the BATTLE command. Please change it to reflect what your new command wants to do!